langs.json

```diff
@@ -2398,5 +2398,15 @@
         "names": ["Zaza", "Dimili", "Dimli", "Kirdki", "Kirmanjki", "Zazaki"],
         "two": "",
         "three": "zza"
+    },
+    {
+        "names": ["Garo"],
+        "two": "",
+        "three": "grt"
+    },
+    {
+        "names": ["Mizo"],
+        "two": "lus",
```

Comment:
Are `two` values supposed to have 2 characters? This is the only one that is not either empty or has two characters.

Reply:
i assumed this was `ISO 639-2` and `ISO 639-3` notation and was hoping for someone with better experience with this repo confirm it. 
Searched now [here](https://github.com/dimagi/langcodes/search?q=two&unscoped_q=two) and looks like it is a bit entangled if i understand this correctly.

The only languages for which `two` is mainly relevant are [these 4 languages](https://github.com/dimagi/langcodes/blob/25518000beeb033be2b3d8a816640757ba6e862a/__init__.py#L13) considering how we handle them during search [here](https://github.com/dimagi/langcodes/blob/ef49439869066525e7dfa8941ececa831983dd15/views.py#L14). So only for these there `two` code would be shown when searching.

Or otherwise `two` is just used for search [here](https://github.com/dimagi/langcodes/blob/ef49439869066525e7dfa8941ececa831983dd15/views.py#L33) which would mean two different notations for the same language.

In either case having "two" defined is mostly for searching and i think having them same is just not relevant. So i think i can just remove that one here.

For improvement, on search, we can check for "three" before "two" since that would be an earlier hit instead of checking for blank value first and then for the valid value.